src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -261,36 +241,39 @@
               <CompletedIa class="mt-6" />
             </template>
 
-            <!-- is this a FE IA pending (overdue)? -->
-            <template v-else-if="filing.isFutureEffectiveIaPending">
-              <FutureEffectivePending :filing=filing class="mt-6" />
+            <!-- is this a completed alteration? -->
+            <template v-else-if="filing.isCompletedAlteration">
+              <CompletedAlteration :filing=filing class="mt-6" />
             </template>
 
-            <!-- is this a FE IA still waiting for effective date/time? -->
-            <template v-else-if="filing.isFutureEffectiveIa">
-              <FutureEffective :filing=filing class="mt-6" />
+            <!-- is this a completed dissolution? -->
+            <template v-else-if="filing.isCompletedDissolution">
+              <CompletedDissolution :filing="filing" class="mt-6" />
             </template>
```

Comment:
O nice, i've not seen one yet. Was there some test data loaded in for this?

Reply:
Thor said he would create some. I have followed up with him.
For this, I temporarily hacked the UI code to display whatever detail component I wanted.

Reply:
Sounds good!

Reply:
Thanks for resolving this conversation. As I mentioned earlier, after this PR is merged I will park this ticket until we find out more what the API will provide. At that time there will be real dissolution filings / entities to look at. No further action needed here atm.